pyproject.toml

```diff
@@ -35,10 +35,10 @@ select = ["ALL"]
 ignore = [
   "C90", # mccabe
   "CPY", # flake8-copyright
-  "EM",  # flake8-errmsg
+  "EM", # flake8-errmsg
```

Comment:
Mind reverting these other formatting changes?

I assume your editor just auto-formatted these and you accidentally committed that without checking? I like to use the `-p` (`--patch`) flag when adding files, so I can pick the hunks to add interactively.

But if it's too annoying to set up your editor in a way that it won't auto-format the file like this, it's fine.